packages/authorization-process/test/addClientPurpose.test.ts

```diff
@@ -117,6 +121,165 @@ describe("addClientPurpose", async () => {
       }),
     });
   });
+
+  it("should write on event-store for the addition of a purpose into a client when the tenant has an active Consumer delegation", async () => {
+    const mockDescriptor: Descriptor = {
+      ...getMockDescriptor(),
+      state: descriptorState.published,
+      interface: getMockDocument(),
+      publishedAt: new Date(),
+    };
+
+    const mockEservice = {
+      ...getMockEService(),
+      descriptors: [mockDescriptor],
+    };
+    const mockConsumerId: TenantId = generateId();
+
+    const mockPurpose: Purpose = {
+      ...getMockPurpose(),
+      eserviceId: mockEservice.id,
+      consumerId: mockConsumerId,
+      versions: [getMockPurposeVersion(purposeVersionState.active)],
+    };
```

Comment:
Explicit the fact that this purpose is not created by the delegate

```suggestion
    const mockPurpose: Purpose = {
      ...getMockPurpose(),
      eserviceId: mockEservice.id,
      consumerId: mockConsumerId,
      versions: [getMockPurposeVersion(purposeVersionState.active)],
      delegationId: undefined
    };
```